samples/algorithms/sudoku-grover/ColoringGroverWithConstraints.qs

```diff
@@ -113,257 +113,66 @@ namespace Microsoft.Quantum.Samples.ColoringGroverWithConstraints {
     ///  1---2
     /// ```
     /// i.e. every vertex is connected to each other.
-    /// Additionally, we require that:
-    ///
-    ///    - vertices 0 and 1 do not get colors 2 and 3.
-    ///    - vertices 2 and 3 do not get colors 3 and 0.
-    ///    - vertices 0 and 2 do not get colors 1 and 3.
-    ///    - vertices 1 and 3 do not get colors 2 and 0.
-    /// This results in edges (vertices that can not be same color):
-    /// `edges = [(1, 0),(2, 0),(3, 0),(3, 1),(3, 2)]`
-    /// This is saying that vertex 1 can not have same color as vertex 0 etc.
-    /// and startingColorConstraints = [(0, 1),(0, 3),(0, 2),(1, 2),(1, 0),
-    ///                    (1, 3),(2, 1),(2, 3),(2, 0),(3, 2),(3, 0),(3, 1)]
-    /// This is saying that vertex 0 is not allowed to have values 1,3,2
-    /// and vertex 1 is not allowed to have values 2,0,3
-    /// and vertex 2 is not allowed to have values 1,3,0
-    /// and vertex 3 is not allowed to have values 2,0,1
-    /// A valid graph coloring solution is: [0,1,2,3]
-    /// i.e. vertex 0 has color 0, vertex 1 has color 1 etc.
     operation ApplyVertexColoringOracle (
-        numVertices : Int, bitsPerColor : Int, edges : (Int, Int)[],
-        startingColorConstraints : (Int, Int)[],
+        nVertices : Int, 
+        bitsPerColor : Int, 
+        edges : (Int, Int)[],
         colorsRegister : Qubit[],
         target : Qubit
     )
     : Unit is Adj + Ctl {
         let nEdges = Length(edges);
-        let nStartingColorConstraints = Length(startingColorConstraints);
-        // we are looking for a solution that:
-        // (a) has no edge with same color at both ends and
-        // (b) has no Vertex with a color that violates the starting color constraints.
+        // we are looking for a solution that has no Vertex with a color that violates the starting color constraints.
```

Comment:
I think for this oracle we need to keep the other comment - we encode the prohibited colors in the search space, so the oracle only looks for a solution that has no edge connecting two vertices of the same color